lib/ember_cli.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module EmberCli
+  def self.enabled?
+    ENV["EMBER_CLI_PROD_ASSETS"] != "0"
+  end
+
+  def self.script_chunks
+    return @@chunk_infos if defined? @@chunk_infos
```

Comment:
Based on what I'm seeing, this info here is computed for every unicorn process the first time this method is called. Would it be helpful to preload this information before forking?